lib/react/server_rendering/bundle_renderer.rb

```diff
@@ -102,6 +103,8 @@ def assets_precompiled?
       def asset_container_class
         if self.class.asset_container_class.present?
           self.class.asset_container_class
+        elsif SeparateServerBundleContainer.compatible? && self.class.dont_use_webpacker_manifest
```

Comment:
dont use webpacker manifest should be the ONLY case for SSR

using the webpack dev server for the server bundle is a flawed approach

I originally supported this for React on Rails, and that was a mistake.